### ISSUE TITLE
lexicons not found.

### ISSUE THREAD

**Issue by Shualite** [MhLiao/MaskTextSpotterV3#12]
When I try to eval IC15 inference results, I find the lexicons dir is not there.

the code below is in evauation/icdar2015/prepare_results.py line 149:
`for i in tqdm(range(1,501)):
        img = 'img_'+str(i)+'.jpg'
        gt_img = 'gt_img_'+str(i)+'.txt'
        if lexicon_type==3:
            # weak
            lexicon_path = '../../lexicons/ic15/new_strong_lexicon/new_voc_img_' + str(i) + '.txt'
            lexicon_fid=open(lexicon_path, 'r')
            pair_list = open('../../lexicons/ic15/new_strong_lexicon/pair_voc_img_' + str(i) + '.txt', 'r')
            pairs = dict()
            for line in pair_list.readlines():
                line=line.strip()
                word = line.split(' ')[0].upper()
                word_gt = line[len(word)+1:]
                pairs[word] = word_gt
            lexicon_fid=open(lexicon_path, 'r')
            lexicon=[]
            for line in lexicon_fid.readlines():
                line=line.strip()
                lexicon.append(line)`

I have visited the icdar competition website:
`Task 4.4: End to End (2015 edition)
Training Set
Training Set Images (88.5MB).- 1000 images obtained with wearable cameras

Training Set Vocabulary (16KB).- Vocabulary of all words (words of 3 characters or longer comprising only letters) appearing in the training set

Training Set Per-image Vocabularies (504KB).- Vocabularies of 100 words per image, comprising the words appearing in the image plus distractors

Training Set Localisation and Transcription Ground Truth (157KB).- 1000 text files with word level localisation and transcription ground truth

Test Set
Test Set Images (43.3MB).- 500 images obtained with wearable cameras. You can submit your results for this Task over the images of the test set through the My Methods section.

Test Set Vocabulary (8KB).- Vocabulary of all words (words of 3 characters or longer comprising only letters) appearing in the test set

Test Set Per-image Vocabularies (248KB).- Vocabularies of 100 words per image, comprising the words appearing in the image plus distractors

Test Set Ground Truth (244Kb). - 500 text files with text localisation bounding boxes for the images of the test set.

Other
 Generic Vocabulary (796KB).- A vocabulary of about 90k words derived from the dataset publicly available here. Please consult [1,2] for further information as well as the disclaimer in the vocabulary file itself.`

But the item "Training Set Per-image Vocabularies" seems not containing 'new_voc_img_*.txt' and 'pair_voc_img_*.txt'.
So I'm confused! Please help me, thank u.

**Comment by MhLiao** [comment]
@Shualite The downloading link of the lexicons is added.

**Comment by DatDoc** [comment]
Hi @MhLiao, may I ask where is the link to download the lexicon? Thank you.
